kcidb/db/argparse.py

```diff
@@ -0,0 +1,200 @@
+import argparse
+import kcidb.orm
+import kcidb.misc
+from kcidb.db import abstract, schematic, mux, \
+    bigquery, postgresql, sqlite, json, null, misc  # noqa: F401
+# import sys
+# import logging
+# import datetime
+# import kcidb.io as io
+# from kcidb.misc import LIGHT_ASSERTS
+
+
+class DBHelpAction(argparse.Action):
+    """Argparse action outputting database string help and exiting."""
+    def __init__(self,
+                 option_strings,
+                 dest=argparse.SUPPRESS,
+                 default=argparse.SUPPRESS,
+                 help=None,
+                 ):
+        super().__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            nargs=0,
+            help=help)
+
```

Comment:
This is the file I am having an error.  See we are importing both argparse and kcidb/argparse. That creates confusion.

Reply:
You have only one argparse in your package where I have multiple. When I import the argparse  This can be solved if we import argparse as ap(or something else). So then it does not confuse with other argparse that we will import from kcidb.